arithmetization/src/main/java/net/consensys/linea/zktracer/module/limits/Keccak.java

```diff
@@ -54,14 +62,16 @@ public String moduleKey() {
 
   @Override
   public void enterTransaction() {
-    this.deployedCodesizes.push(new ArrayList<>(5));
+    this.deployedCodeSizes.push(new ArrayList<>(8));
```

Comment:
Why is the ArrayList given a fixed size ? Where does it come from ?

Reply:
IIRC they are minimum sizes to achieve:
- avoiding to copy the underlying array  too often as you add stuff in it (with the expectation that 8 items should be enough most of the time)
- not allocating too much space either.

Reply:
yes, but the 8, 100, 4, where this values come from ? @powerslider  ?